src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/StringParsingHelpers.Statistics.cs

```diff
@@ -443,9 +443,9 @@ internal static IPInterfaceStatisticsTable ParseInterfaceStatisticsTableFromFile
             }
         }
 
-        private static uint ParseUInt64AndClampToUInt32(string value)
+        private static long ParseUInt64AndClampToInt64(string value) 
         {
-            return (uint)Math.Min(uint.MaxValue, ulong.Parse(value));
+            return (long)Math.Min((ulong)long.MaxValue, ulong.Parse(value)); 
```

Comment:
Should these specify invariant culture? I see it was missed before. I assume this file is not in localized format.